x-pack/plugins/data_visualizer/public/application/index_data_visualizer/hooks/esql/use_esql_overall_stats_data.ts

```diff
@@ -242,18 +241,26 @@ export const useESQLOverallStatsData = (
         // For doc count chart, we want the full base query without any limit
         const esqlBaseQuery = searchQuery.esql;
 
+        // dropNullColumns wills all columns are empty if limti 0
```

Comment:
Typos here - `limti`. Not sure what you mean by `dropNullColumns wills...`

Reply:
Removed here https://github.com/elastic/kibana/pull/179098/commits/681ec1acf2c1ad1f8181eb87957382bef8bb5fd5